src/js/containers/router/RouterRoutes.jsx

```diff
@@ -61,7 +61,7 @@ const routes = {
             parent: '/award',
             component: (cb) => {
                 require.ensure([], (require) => {
-                    cb(require('containers/award/AwardContainer').default);
+                    cb(require('containers/awardV2/AwardV2Container').default);
                 });
```

Comment:
Blown away we have our own custom router... Really interested by that and impressed! Would love to hear the rationale around that sometime!